lib/youtube-dl.js

```diff
@@ -238,6 +238,10 @@ ytdl.exec = function exec(url, args, options, callback) {
  * @returns {Object}
  */
 function parseInfo(data) {
+  // Youtube-dl might return just an url as a string when using the "-g" or "--get-url" flag
+  if (typeof data === "string" && data.startsWith("http"))
```

Comment:
not a requisite but appreciated if you can extract this into a function

```js
const isString = str => typeof str === "string"
```

and then use the function there and the rest of the places, like [here](https://github.com/przemyslawpluta/node-youtube-dl/blob/113ed2a1d61e434f35d7fa173770043c80704ddd/lib/youtube-dl.js#L191).

Reply:
Done in e3f4a9d